step/src/test/java/com/google/sps/CalendarTest.java

```diff
@@ -1,68 +1,68 @@
 // Copyright 2019 Google LLC
-//
+
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-//
+
 //     https://www.apache.org/licenses/LICENSE-2.0
-//
+
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps;
+// package com.google.sps;
 
-import java.lang.IllegalStateException;
-import java.util.List;
-import java.util.ArrayList;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.Before;
-import org.junit.After;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
-import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
-import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
-import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
-import com.google.api.client.auth.oauth2.Credential;
-import com.google.api.client.http.HttpTransport;
-import com.google.api.client.http.HttpRequestInitializer;
-import com.google.api.client.http.javanet.NetHttpTransport;
-import com.google.api.client.json.JsonFactory;
-import com.google.api.client.json.jackson2.JacksonFactory;
-import com.google.api.services.calendar.Calendar;
-import com.google.api.services.calendar.Calendar.Builder;
-import com.google.api.services.calendar.model.Event;
-import com.google.api.client.googleapis.testing.services.MockGoogleClient;
-import com.google.api.client.googleapis.testing.auth.oauth2.MockGoogleCredential;
-import org.mockito.runners.MockitoJUnitRunner;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.mockito.Mockito.anyString;
-import org.mockito.ArgumentMatchers;
+// import java.lang.IllegalStateException;
```

Comment:
There's no need to comment out this entire file. If you want to stop one of these tests from running, just comment out the @Test notation on the test

Reply:
I am sorry. I should uncomment before pushing. But sure, thank you for the recommendation!